litex/build/lattice/programmer.py

```diff
@@ -185,3 +185,18 @@ def load_bitstream(self, bitstream_file):
             "--freq", str(self.frequency_khz),
             bitstream_file
         ])
+
+# EcpprogProgrammer -------------------------------------------------------------------------------
+
+class EcpprogProgrammer(GenericProgrammer):
+    """ECPDAP allows you to program ECP5 FPGAs and attached SPI flash using FTDI based JTAG probes
```

Comment:
Should this comment say `ecpprog` instead of `ECPDAP`.

Reply:
Good catch, thanks! (and thanks for `ecpprog`)